### PR TITLE
Added configurable tweet by

### PR DESCRIPTION
**Pull request by tallowen**

### CODE REVIEW

**Comment by tallowen** [comment]
This is this fixes issue 4 https://github.com/mozilla/SocialShare/issues/4

File: README.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ Insert the following code to get it to run
 
 Other resources (images, fonts) should be reached from their own media folder.
 
+### Options
+
+Name         | Value 
+------------ | ------------- 
+data-size    | Controls the size of the share iconts
```
Comment (fwenzel) [inline README.md]: "icons" Also, write what I am allowed to put in there.

**Comment by fwenzel** [comment]
looks good. Does it still default to firefox?

**Comment by tallowen** [comment]
Yup. Defaults to firefox.

**Comment by tofumatt** [comment]
r+wc then!

**Pull request merged**